osquery/remote/http_client.cpp

```diff
@@ -30,11 +30,10 @@ void Client::callNetworkOperation(std::function<void()> callback) {
   callback();
 
   {
-    boost::system::error_code rc;
-    ioc_.run(rc);
-    ioc_.reset();
```

Comment:
[Deprecated and replaced](https://live.boost.org/doc/libs/1_86_0/doc/html/boost_asio/reference/io_context/reset.html)